### ISSUE TITLE
Pulp Test failes to build RowList

### ISSUE THREAD

**Issue by guygastineau** [exercism/purescript#177]
`pulp test` fails:

```
Error found:
in module Type.Data.RowList
at bower_components/purescript-prelude/src/Type/Data/RowList.purs:8:26 - 8:33 (line 8, column 26 - line 8, column 33)

  Could not match kind
                
    Type -> Type
                
  with kind
        
    Type
        

while checking that type RowList
  has kind Type
in type constructor RLProxy
```

I can't seem to find anything about this on the internet... :/  I don't where the conflict is in the `RowList` dependency, but obviously the kinds `* -> *` and `*` are not the same lol.

I wanted to come back to purescript a bit, so I unpaused the track, but I can't get my old passing examples to build.  purescript, pulp, and bower are all fresh installs (from a couple hours ago).  I tried to build the next exercise in the queue for myself (pangram), but I get the same build error everytime.

I would really appreciate any help at all getting this to build.  Thank you.

**Comment by guygastineau** [comment]
Update: I got it building by abandoning bower and pulp altogether.

1. I installed `spago`
2. I `spago init`ed
3. I `spago test`ed

I guess I will look into @connorads' work on #127, and I will try to help get that ready for prime time if I am able.  Unfortunately, I guess those changes won't actually take effect here until v3 launches, but it will eventually be impactful ?

Anyway, at least I can function using spago to replace bower directly, but I will need to tell the mentors that is how I built it I guess.
